### PR TITLE
Add base-2/10 switch and iteration arg to bytes_and_flops

### PR DESCRIPTION
**Pull request by arghdos**
Change-Id: I767002da3003b0b15e07abefcbd9faf6629e5043

### CODE REVIEW

File: benchmarks/bytes_and_flops/main.cpp [hunk 1]
```diff
@@ -105,17 +119,26 @@ int main(int argc, char* argv[]) {
     return 0;
   }
 
+  if ((B != 2) && (B != 10)) {
+    printf("B must be one of 2,10\n");
+    return 0;
```
Comment (dalg24) [inline benchmarks/bytes_and_flops/main.cpp:124]: I love that we return success error code.  (not asking you to change I see it is consistent with what was there before)

File: benchmarks/bytes_and_flops/main.cpp [hunk 2]
```diff
@@ -105,17 +119,26 @@ int main(int argc, char* argv[]) {
...
+  }
+
+  if (I < 1) {
+    printf("I must be >= 1\n");
+  }
```
Comment (dalg24) [inline benchmarks/bytes_and_flops/main.cpp:129]:
```suggestion
  if (I < 1) {
    printf("I must be >= 1\n");
    return 0;
  }
```

File: benchmarks/bytes_and_flops/main.cpp [hunk 3]
```diff
@@ -72,6 +72,10 @@ int main(int argc, char* argv[]) {
     printf("  T:   team size\n");
     printf(
         "  S:   shared memory per team (used to control occupancy on GPUs)\n");
+    printf(
+        "  B:   units for reported memory bandwidths (2=GiB, 10=GB, "
+        "default=2)");
+    printf("  I:   iterations of the kernel to time over(default=10)");
```
Comment (dalg24) [inline benchmarks/bytes_and_flops/main.cpp:78]:
```suggestion
    printf("  I:   iterations of the kernel to time over (default=10)\n");
```

File: benchmarks/bytes_and_flops/main.cpp [hunk 4]
```diff
@@ -72,6 +72,10 @@ int main(int argc, char* argv[]) {
     printf("  T:   team size\n");
     printf(
         "  S:   shared memory per team (used to control occupancy on GPUs)\n");
+    printf(
+        "  B:   units for reported memory bandwidths (2=GiB, 10=GB, "
+        "default=2)");
```
Comment (dalg24) [inline benchmarks/bytes_and_flops/main.cpp:77]:
```suggestion
        "default=2)\n");
```

File: benchmarks/bytes_and_flops/main.cpp [hunk 5]
```diff
@@ -48,19 +48,19 @@
...
 extern template void run_stride_unroll<int64_t>(int, int, int, int, int, int,
-                                                int, int);
+                                                int, int, int, int);
 
 int main(int argc, char* argv[]) {
   Kokkos::initialize();
 
   if (argc < 10) {
-    printf("Arguments: N K R D U F T S\n");
+    printf("Arguments: N K R D U F T S B\n");
```
Comment (dalg24) [inline benchmarks/bytes_and_flops/main.cpp:63]:
```suggestion
    printf("Arguments: N K R D U F T S B I\n");
```

**Review by dalg24 — CHANGES_REQUESTED** [review]

**Review by Rombur — APPROVED** [review]

**Review by crtrott — APPROVED** [review]

**Pull request merged**